content/LGinC/drone-portainer/index.md

```diff
@@ -0,0 +1,64 @@
+---
+date: 2020-07-16T00:00:00+00:00
+title: "Portainer Stack Deploy"
+author: "LGinC"
+tags: [ deploy ]
+repo: "turinge/drone-portainer"
```

Comment:
The repo doesn't match the folder of this file, so which one is the right one now?

Reply:
I had added it into readme

Reply:
But the path to this markdown document still doesn't match the repo value.

Reply:
has updated path match to repo

Reply:
But https://github.com/turinge/drone-portainer results in a 404, so you added a wrong reference. repo is the name of the Git repository, image ist the name of the Docker image.

Reply:
update it.
[lginc/drone-portainer](https://hub.docker.com/r/lginc/drone-portainer)
[https://github.com/LGinC/drone-portainer](https://github.com/LGinC/drone-portainer)